pkg/scalers/azure_eventhub_scaler_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"os"
 	"testing"
 
+	kedav1alpha1 "github.com/kedacore/keda/v2/apis/keda/v1alpha1"
```

Comment:
Could you split the imports in groups? I mean, split imports into "golang", "dependencies" and "internals" please

Reply:
There's a linter `gci` which you can add to golangci that does this, should I add it?

Reply:
IDK how it works TBH, but if that linter checks that all imports are split in those 3 groups, it'd be nice

Reply:
I mean, there are 3 cases in this PR, maybe it's easier just update them manually in this PR and adding the linter in another. It's a real possibility that there are more places in the code that it's not correctly split, so adding the linter in this PR could require a lot of changes

Reply:
I will make the changes. Meanwhile, could you check the e2e tests? I am not sure what changed in the Github actions / the e2e script but the tests are not running.

Reply:
I think that we don't have yet `AD Workload Identity` deployed in the cluster, but maybe we can have the e2e tests about it there disabled till we solve the problem of having the dependencies. (I hope that soon)